carbon-design-system/carbon
# fix(react): uishell headermenu detect active descendants

#### Changelog

**Changed**

- Updated the React HeaderMenu in UIShell so that it detects active descendant links when more than two levels of navigation are in the page heirarchy

#### Testing / Reviewing

- Create HeaderNavigation with three levels of nesting.
- Set a third-level link to 'isActive'
- Notice that the parent link in the main navigation is marked as active when the dropdown is not opened.

Related to https://github.com/carbon-design-system/carbon/pull/15027
